app/namespace.go

```diff
@@ -1425,6 +1435,66 @@ func NewNamespaceCommand(getNamespaceClientFn GetNamespaceClientFn) (CommandOut,
 		},
 	})
 
+	// Note: the following commands are depend on previous commands. Please don't add more commands in the middle
```

Comment:
this seems somewhat confusing / brittle. Is there a better way to programmatically search and find the export command we care about by maybe checking the .Name field?

Reply:
Cool